compiler/src/dotty/tools/dotc/quoted/QuoteUnpickler.scala

```diff
@@ -0,0 +1,23 @@
+package dotty.tools.dotc.quoted
+
+import dotty.tools.dotc.core.tasty._
+
+object QuoteUnpickler {
```

Comment:
Should this be in `tasty` instead of `quoted`?

Reply:
We already have `TastyUnpickler` that is used inside on the `QuoteUnpickler`, it might be confusing.

Reply:
I don't see it as confusing.

Reply:
Then I'll change it